src/slurm_plugin/common.py

```diff
@@ -64,7 +68,7 @@ def _clear_failed_nodes(self):
         """Clear and reset failed nodes list."""
         self.failed_nodes = []
 
-    def add_instances_for_nodes(self, node_list, launch_batch_size, update_node_address=True):
+    def add_instances_for_nodes(self, node_list, launch_batch_size=100, update_node_address=True):
```

Comment:
I'd avoid setting these defaults here (same for the functions below) so that we always force the caller to specify the value. Default can be set in the caller and read from config.